### ISSUE TITLE
_tkinter.TclError: image "pyimage3" doesn't exist

### ISSUE THREAD

**Issue by UniquePhilosophy** [TomSchimansky/TkinterMapView#1]
Thought I'd try to implement your TkinterMapView into my program today and as soon as I tried to display an instance of TkinterMapView:

`    agency_map = TkinterMapView(map_window, width=1920, height=1080, corner_radius=0)
    agency_map.grid(row=0, column=0)
`

I got the error mentioned in the title, referring to a series of tkintermapview functions.
Has support ended for this library?

Thanks,
JR

**Comment by TomSchimansky** [comment]
Maybe you can post your full code, and the full error message. Are you on Windows or macOS?

**Comment by UniquePhilosophy** [comment]
I can't post the full code, but I will post everything I think is relevant:

**Function that loads new window and creates and places instance of map** (I tried again with your dimensions etc. to make sure it wasn't the way I was constructing the map):
`
def show_map():
    map_window = Tk()
    map_window.title('Map View')
    map_window.geometry('1920x1080+0+0')

    agency_map = TkinterMapView(map_window, width=800, height=600, corner_radius=0)
    agency_map.place(relx=0.5, rely=0.5, anchor=CENTER)
`

**One line code importing the library** (After 'pip install tkintermapview'):

`
from tkintermapview import TkinterMapView
`

That's as simple as it gets really, the full error is this:

_Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\Jord\AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 1892, in __call__
    return self.func(*args)
  File "C:\Users\Jord\AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 814, in callit
    func(*args)
  File "C:\Users\Jord\PycharmProjects\superposer_v0.0.1\venv\lib\site-packages\tkintermapview\map_widget.py", line 361, in update_canvas_tile_images
    canvas_tile.set_image(image)
  File "C:\Users\Jord\PycharmProjects\superposer_v0.0.1\venv\lib\site-packages\tkintermapview\canvas_tile.py", line 33, in set_image
    self.draw(image_update=True)
  File "C:\Users\Jord\PycharmProjects\superposer_v0.0.1\venv\lib\site-packages\tkintermapview\canvas_tile.py", line 56, in draw
    self.canvas_object = self.map_widget.canvas.create_image(canvas_pos_x,
  File "C:\Users\Jord\AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 2790, in create_image
    return self._create('image', args, kw)
  File "C:\Users\Jord\AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 2776, in _create
    return self.tk.getint(self.tk.call(
_tkinter.TclError: image "pyimage3" doesn't exist_

**Comment by TomSchimansky** [comment]
I cannot reproduce this, maybe you can try it without using a virtual environment. And are you using the latest version 1.1?

**Comment by UniquePhilosophy** [comment]
Hi Tom,

I'll answer your questions first!

1) I don't think I'm using a virtual environment: I'm using PyCharm in Windows to write to .py files directly on my drives.
2) I am using TkinterMapView version 1.1.

I've created a test file which just contains the script:

`
from tkinter import *
from tkinter import ttk
from tkintermapview import TkinterMapView

root = Tk()
root.title('Test')
root.geometry('1220x880+0+0')

agency_map = TkinterMapView(root, width=800, height=600, corner_radius=0)
agency_map.place(relx=0.5, rely=0.5, anchor=CENTER)

root.mainloop()
`

And it works fine...

Is it possible that importing TkinterMapView has messed with my imports? The test program has three imports (* from tkinter, ttk and TkinterMapView) but since I haven't used 'ttk', it is showing as greyed out in my IDE.

In the **real** program, however, I have several imports (* from tkinter, ttk, mysql.connector, TkinterMapView and 'form_fields' & 'settings' (two of my own source code files). The key difference I have noticed is that the line 'from tkinter import *' has been greyed out - I think this occurred after importing TkinterMapView.

**Comment by UniquePhilosophy** [comment]
Hi Tom,

I'll answer your questions first!

1) I don't think I'm using a virtual environment: I'm using PyCharm in Windows to write to .py files directly on my drives.
2) I am using TkinterMapView version 1.1.

I've created a test file which just contains the script:

`
from tkinter import *
from tkinter import ttk
from tkintermapview import TkinterMapView

root = Tk()
root.title('Test')
root.geometry('1220x880+0+0')

agency_map = TkinterMapView(root, width=800, height=600, corner_radius=0)
agency_map.place(relx=0.5, rely=0.5, anchor=CENTER)

root.mainloop()
`

And it works fine...

Is it possible that importing TkinterMapView has messed with my imports? The test program has three imports (* from tkinter, ttk and TkinterMapView) but since I haven't used 'ttk', it is showing as greyed out in my IDE.

In the **real** program, however, I have several imports (* from tkinter, ttk, mysql.connector, TkinterMapView and 'form_fields' & 'settings' (two of my own source code files). The key difference I have noticed is that the line 'from tkinter import *' has been greyed out - I think this occurred after importing TkinterMapView.

**Comment by UniquePhilosophy** [comment]
Hi Tom,

I'll answer your questions first!

1) I don't think I'm using a virtual environment: I'm using PyCharm in Windows to write to .py files directly on my drives.
2) I am using TkinterMapView version 1.1.

I've created a test file which just contains the script:

`
from tkinter import *
from tkinter import ttk
from tkintermapview import TkinterMapView

root = Tk()
root.title('Test')
root.geometry('1220x880+0+0')

agency_map = TkinterMapView(root, width=800, height=600, corner_radius=0)
agency_map.place(relx=0.5, rely=0.5, anchor=CENTER)

root.mainloop()
`

And it works fine...

Is it possible that importing TkinterMapView has messed with my imports? The test program has three imports (* from tkinter, ttk and TkinterMapView) but since I haven't used 'ttk', it is showing as greyed out in my IDE.

In the **real** program, however, I have several imports (* from tkinter, ttk, mysql.connector, TkinterMapView and 'form_fields' & 'settings' (two of my own source code files). The key difference I have noticed is that the line 'from tkinter import *' has been greyed out - I think this occurred after importing TkinterMapView.

**Comment by TomSchimansky** [comment]
Sorry, I deleted your post, because it showed up three times for me, but looks like it was just a ui bug. Anyway your post was:

Hi Tom,

I'll answer your questions first!

I don't think I'm using a virtual environment: I'm using PyCharm in Windows to write to .py files directly on my drives.
I am using TkinterMapView version 1.1.
I've created a test file which just contains the script:

```python
from tkinter import *
from tkinter import ttk
from tkintermapview import TkinterMapView

root = Tk()
root.title('Test')
root.geometry('1220x880+0+0')

agency_map = TkinterMapView(root, width=800, height=600, corner_radius=0)
agency_map.place(relx=0.5, rely=0.5, anchor=CENTER)

root.mainloop()
```

And it works fine...

Is it possible that importing TkinterMapView has messed with my imports? The test program has three imports (* from tkinter, ttk and TkinterMapView) but since I haven't used 'ttk', it is showing as greyed out in my IDE.

In the real program, however, I have several imports (* from tkinter, ttk, mysql.connector, TkinterMapView and 'form_fields' & 'settings' (two of my own source code files). The key difference I have noticed is that the line 'from tkinter import *' has been greyed out - I think this occurred after importing TkinterMapView.
